lib/appcanary/assert.rb

```diff
@@ -0,0 +1,51 @@
+require "json"
+
+module Appcanary
+  class << self
+    def frequencies(arr)
+      arr.inject({}) do |freqs, i|
+        freqs[i] ||= 0
+        freqs[i] += 1
+        freqs
+      end
+    end
+
+    def criticalities(response)
+      frequencies response["included"]
+        .map { |vuln| vuln["attributes"]["criticality"] }
+    end
+
+    def vulnerable?
+      ship_gemfile :check do |response|
+        !! response["meta"]["vulnerable"]
+      end
+    end
+
+    def am_I_critically_fucked?
+      # you're fucked if you have critical vulnerabilities
+      ship_gemfile :check do |response|
+        if response["meta"]["vulnerable"]
```

Comment:
Minor style, try instead:

```ruby
if vulnerable?(response)
```

i.e. have a method that checks ["meta"]["vulnerable"] for you :).